C++/Stack/MaximumAreaHistogram.cpp

```diff
@@ -0,0 +1,56 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+int largest_area_histogram(int arr[],int n)
+{
+    int max_area = 0;
+    int rb[n]; // nse on the right

```

Comment:
The naming of variables is not understandable by the user. Please rename rb and lb to some useful names.

Reply:
Sure